bindings/wasm/src/did/wasm_core_document.rs

```diff
@@ -110,6 +114,8 @@ impl WasmCoreDocument {
   }
 
   /// Sets the `alsoKnownAs` property in the DID document.
+  ///
+  /// Warning: Improper use can lead to broken URI dereferencing.
```

Comment:
I don't know if most users would know what this really means. Can we be more concrete as to what users should avoid doing?